README.md

```diff
@@ -1 +1,14 @@
-# types
\ No newline at end of file
+# longhorn types
+
+## Dependencies
+
+* protoc 24.3
+* protoc-gen-go 1.31.0
+* protoc-gen-go-grpc 1.3.0
+* python protobuf 4.24.3
```

Comment:
nit: Where should developers make changes if they want to update?

Reply:
Sorry, I add `Dockerfile.dapper*` to `.gitignore`, so we don't see it here. I will add it and force push again. We can change it in `Dockerfile.dapper` in the future.

Reply:
I add `Dockerfile.dapper`. There are some variables can change the version. Thanks.